### PR TITLE
 New Package: runit-backlight-20200420

### PR DESCRIPTION
**Pull request by StellarOrbit**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by abenson** [comment]
Drop the commits related to `NetworkManager-fortisslvpn`

File: srcpkgs/backlight/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template for backlight-runit from Artix LInux
+pkgname=backlight
+version=2020420
+revision=1
+short_desc="service for backlight saving across reboots"
+maintainer="Twilighttony <starrbeartony14@zoho.com>"
+license="BSD"
+homepage="https://gitea.artixlinux.org/artixlinux/packages-runit/src/branch/master/backlight-runit/x86_64/community"
+backlight_homedir="/etc/sv/backlight"
+#make_dirs="/etc/sv/backlight"
```
Comment (abenson) [inline srcpkgs/backlight/template:10]: Remove this

File: srcpkgs/backlight/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template for backlight-runit from Artix LInux
+pkgname=backlight
+version=2020420
+revision=1
+short_desc="service for backlight saving across reboots"
+maintainer="Twilighttony <starrbeartony14@zoho.com>"
+license="BSD"
+homepage="https://gitea.artixlinux.org/artixlinux/packages-runit/src/branch/master/backlight-runit/x86_64/community"
+backlight_homedir="/etc/sv/backlight"
```
Comment (abenson) [inline srcpkgs/backlight/template:9]: This does nothing since there's no system user created

File: srcpkgs/backlight/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Template for backlight-runit from Artix LInux
+pkgname=backlight
+version=2020420
+revision=1
+short_desc="service for backlight saving across reboots"
+maintainer="Twilighttony <starrbeartony14@zoho.com>"
+license="BSD"
```
Comment (abenson) [inline srcpkgs/backlight/template:7]: Use the SPDX identifier, i.e. `BSD-2-Clause`, `BSD-3-Clause`, etc

**Comment by Twilighttony** [comment]
> Drop the commits related to `NetworkManager-fortisslvpn`

Not sure why those were added here since I changed branches and did a PR

File: srcpkgs/backlight/template [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Template for backlight-runit from Artix LInux
```
Comment (abenson) [inline srcpkgs/backlight/template:1]: drop the "from Artix LInux"

File: srcpkgs/backlight/template [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+pkgname=backlight
```
Comment (abenson) [inline srcpkgs/backlight/template:2]: I'd recommend renaming to `runit-backlight`, following `runit-iptables`.

File: srcpkgs/backlight/template [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+version=2020420
+revision=1
+short_desc="service for backlight saving across reboots"
```
Comment (abenson) [inline srcpkgs/backlight/template:5]: Run `xlint` (from `xtools`) on the template

**Comment by Duncaen** [comment]
What is the point of making this a fake service?

**Comment by ericonr** [comment]
I think the consensus was to not package exec pause services any more? The first part here can happen in core-services instead, shut down is the complicated one.

**Comment by Twilighttony** [comment]
> What is the point of making this a fake service?

I would say the purpose of this service is so that when you reboot, your screen brightness can be maintained across the reboots, rather than just resetting itself back to defaults.

**Comment by Twilighttony** [comment]
> I think the consensus was to not package exec pause services any more? The first part here can happen in core-services instead, shut down is the complicated one.

I see. So what would you recommend for making this more suitable for void linux? Having screen brightness that doesn't reset across reboots was a big sticking point for me with other runit based distros.

**Comment by ericonr** [comment]
I'm not sure. Maybe provide a small script around a tool like `brightnessctl` that can be added to `rc.local`?

**Pull request closed without merging**